config/aeotec/zw111.xml

```diff
@@ -361,4 +362,10 @@ The Nano Dimmer is also a security Z-Wave plus device and supports Over The Air
       <Group index="4" label="Control Key2" max_associations="5"/>
     </Associations>
   </CommandClass>
+  <CommandClass id="32" name="COMMAND_CLASS_BASIC">
```

Comment:
Remove the `name="COMMAND_CLASS_BASIC"` part

Reply:
Any reason why this should be removed?

Reply:
`The attribute 'name' is not allowed.` per the check.

Reply:
Thanks for your help. Updated.